src/form-field/internal.tsx

```diff
@@ -115,7 +117,10 @@ export default function InternalFormField({
       {(constraintText || errorText) && (
         <div className={styles.hints}>
           {errorText && (
-            <FormFieldError id={slotIds.error} errorIconAriaLabel={i18nStrings?.errorIconAriaLabel}>
+            <FormFieldError
```

Comment:
could we move this inside <FormFieldError> instead, that way we don't have to provide a separate string in TagEditor as well (where this internal component is reused)

Reply:
yes (and that can stay), but moving the context usage into the subcomponent (and I guess making `FormFieldError.errorIconAriaLabel` optional) means that _we_ don't have to provide a new default for TagEditor